### ISSUE TITLE
ref_model is none when use lora?

### ISSUE THREAD

**Issue by akk-123** [lvwerra/trl#365]
when use lora, [ref_model is none](https://github.com/lvwerra/trl/blob/main/trl/trainer/ppo_trainer.py#L210)
if self.ref_model is none, how to compute logits?
```python
ref_logprobs, _, _, _ = self.batched_forward_pass(self.ref_model, queries, responses, model_inputs)
```

**Comment by younesbelkada** [comment]
Hi @akk-123 
In this case, the ref_logprobs are retrieved inside this block: https://github.com/lvwerra/trl/blob/31cc361d1749bb385e205b211f0c2f1f51e7bd26/trl/trainer/ppo_trainer.py#L609
